rasa_nlu/model.py

```diff
@@ -255,7 +255,7 @@ def ensure_model_compatibility(metadata):
 
         model_version = metadata.get("rasa_nlu_version", "0.0.0")
         if version.parse(model_version) < version.parse("0.12.0a2"):
-            raise UnsuportedModelError("The model version is to old to be "
+            raise UnsupportedModelError("The model version is to old to be "
                                        "loaded by this Rasa NLU instance. "
```

Comment:
Do you mind fixing the indentation here?

Reply:
done